src/runtime_src/core/include/xgq_cmd_ert.h

```diff
@@ -137,6 +137,20 @@ struct xgq_cmd_config_start {
 	uint32_t resvd:12;
 };
 
+struct xgq_cmd_clock_calib {
+	struct xgq_cmd_sq_hdr hdr;
+};
+
+struct xgq_cmd_access_valid {
+	struct xgq_cmd_sq_hdr hdr;
+};
+
+struct xgq_cmd_data_integrity {
+	struct xgq_cmd_sq_hdr hdr;
+
+	uint32_t rw_count;
+	uint32_t draft;
+};
```

Comment:
Please comment document these structs.

Reply:
> Please comment document these structs.
definition added, please help me review it, thanks!